### DISCUSSION TITLE
Can't convert tasks to issues, error message unclear

### DISCUSSION THREAD

**Discussion by lutzky** [community/community#4556] [Projects and Issues]
On https://github.com/babybuddy/babybuddy/issues/229, some of the tasks have already been converted to issues; some, such as https://github.com/babybuddy/babybuddy/issues/230, by myself (even though I'm not the owner). However, after editing the issue a few times to add more tasks, now trying to convert tasks to issues gives me this error message:

> We are unable to convert the task to an issue at this time. Please try again.

It's been that way for a few days. I'm *guessing* that editing the issue confused it. Is this behavior a bug, or does the error message just need to be more specific?

**Comment by AlexFBP** [comment]
Had you check if the target repo for the new Issue is an archived one?
(I have this scenario :arrow_up: but I got a 404 screen after creating the issue )

**Reply by lutzky** [reply]
Definitely not an archived repo, https://github.com/babybuddy/babybuddy

**Comment by metcalfc** [comment]
Anyone have an update here? I get the same. I can't tell if its a permissions issue or an outage. It doesn't seem to ever work for me. Which is odd since its demo'd as a core part of the imagined workflow.

**Comment by adriangroch** [comment]
I'm running into the same problem. Clicking "Convert to issue" was creating new issues as expected, until I deleted one of these issues. Now when attempting to convert, I'm getting the `We are unable to convert the task to an issue at this time. Please try again.` toast notification.

**Comment by ximon** [comment]
A colleage encountered this when in a team with triage permission level, changing this to write allowed it to work - surely this should work for triage as the user can manually create issues?

**Comment by ndeeH** [comment]
Worked just one time and then fails for every other task. 
Any ideas? Very annoying!
